compiler/cpp/src/thrift/generate/t_rs_generator.cc

```diff
@@ -2411,6 +2415,11 @@ string t_rs_generator::struct_to_invocation(t_struct* tstruct, const string& fie
   return args.str();
 }
 
+void t_rs_generator::render_args_struct(t_function* tfunc) {
+    string args_struct_name = service_call_args_struct_name(tfunc);
```

Comment:
I believe you can simply do:

`string args_struct_name(service_call_args_struct_name(tfunc);`

Reply:
Ah thanks, good point! C++ is not my mother tongue 😄